[quant] Factoring out serialization versions

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #43199 [quant] conv_transpose graph patterns
* #40371 [quant] ConvTranspose1d / ConvTranspose2d
* #40370 [quant] conv_transpose1d / conv_transpose2d
* #40360 [quant] conv_transpose1d_prepack / conv_transpose1d_unpack
* #40351 [quant] conv_transpose2d_prepack/conv_transpose2d_unpack
* #39714 [quant] Prep for conv_transpose packing
* **#43183 [quant] Factoring out serialization versions**

Differential Revision: [D23181621](https://our.internmc.facebook.com/intern/diff/D23181621)

Commit:
[quant] Factoring out serialization versions

[ghstack-poisoned]

z-a-f commented:
I created the issue describing the discussion we have: #43168

Commit:
Update on "[quant] Factoring out serialization versions"


Differential Revision: [D23181621](https://our.internmc.facebook.com/intern/diff/D23181621)

[ghstack-poisoned]

jerryzh168 commented on aten/src/ATen/native/quantized/cpu/serialization_versions.h:
```diff
@@ -0,0 +1,192 @@
+#pragma once
```
I feel the name of the file is not very intuitive, why not just call it `serialization`?

Commit:
Update on "[quant] Factoring out serialization versions"


Differential Revision: [D23181621](https://our.internmc.facebook.com/intern/diff/D23181621)

[ghstack-poisoned]

Commit:
Update on "[quant] Factoring out serialization versions"


Differential Revision: [D23181621](https://our.internmc.facebook.com/intern/diff/D23181621)

[ghstack-poisoned]
